cloud_run_service_multiple_regions/main.tf

```diff
@@ -2,17 +2,15 @@
 
 resource "google_project_service" "compute_api" {
   provider = google-beta
-  project                    = "my-project-name"
   service                    = "compute.googleapis.com"
-  disable_dependent_services = true
+  disable_dependent_services = false
   disable_on_destroy         = false
 }
 
 resource "google_project_service" "run_api" {
   provider = google-beta
-  project                    = "my-project-name"
```

Comment:
Should we add:

```
data "google_project" "project" {
}
```

Reply:
We only need the data source workaround for resources that cannot infer project_id from the provider which is mostly `project_iam_*` resources. We can also update https://cloud.google.com/docs/terraform/basic-commands to call out adding the provider config so it is consistent on c.g.c.

Reply:
OK, sounds good. Thanks!

I'll start a CL for that.